index.html

```diff
@@ -26,32 +22,42 @@
       crossorigin=""
     ></script>
   </head>
-  <body>
-    <div class="container">
-      <div class="heading">
-        <span>Weather</span>
-        <div id="todays-date"></div>
+  <body class="min-w-96 p-0 m-0 h-100svh bg-white">
+    <div class="container overflow-hidden p-4">
+      <div class="heading text-[black] mb-4">
+        <span class="font-sans font-bold text-5xl uppercase tracking-wide">
+      Weather</span>
+        <div id="todays-date" class="h-3.5 text-xl font-sans"></div>
       </div>
-      <div class="major-cities-scroll">
-        <a href="#">Johannesburg</a>
-        <a href="#">Cape Town</a>
-        <a href="#">London</a>
-        <a href="#">Los Angeles</a>
-        <a href="#">New York</a>
-        <a href="#">Toronto</a>
-        <a href="#">Melbourne</a>
+      <div class="major-cities-scroll bg-[white] overflow-auto whitespace-nowrap mb-6">
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">Johannesburg</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">Cape Town</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">London</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">Los Angeles</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">New York</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">Toronto</a>
+        <a class="inline-block text-black text-center no-underline p-3.5 hover:bg-blue-400" href="#">Melbourne</a>
```

Comment:
Suggested padding that might be closer to original: `p-3`, other than custom styles or arbitrary values.